website/client/src/components/groups/questDetailModal.vue

```diff
@@ -0,0 +1,450 @@
+<template>
+  <b-modal
+    id="quest-detail-modal"
+    title="Empty"
+    size="md"
+    :hide-footer="true"
+    :hide-header="true"
+    :modal-class="dialogClass"
+  >
+    <div class="dialog-close">
+      <close-icon @click="close()" />
+    </div>
+    <h2 class="text-center textCondensed">
+      {{ selectMode ? $t('selectQuest') : $t('questDetailsTitle') }}
+    </h2>
+    <div class="quest-panel" v-if="selectMode">
+      <div class="quest-panel-header">
+        <h3>
+          {{ $t('yourQuests') }}
+        </h3>
+        <div class="sort-by">
+          <span class="dropdown-label">{{ $t('sort') }}</span>
+          <select-translated-array
+            :right="true"
+            :items="['quantity', 'AZ']"
+            :value="sortBy"
+            class="inline"
+            :inline-dropdown="false"
+            @select="sortBy = $event"
+          />
+        </div>
+      </div>
+
+      <div class="quest-items">
+        <!-- eslint-disable vue/no-use-v-if-with-v-for -->
+        <div
+          v-for="item in questsInfoList"
+          :key="item.key"
+          class="quest-col"
+          @click="selectQuest(item)"
+        >
+          <item
+            :key="item.key"
+            :item="item"
+            :item-content-class="item.class"
+          >
+            <countBadge
+              slot="itemBadge"
+              :show="item.amount !== 1"
+              :count="item.amount"
+            />
+            <template
+              slot="popoverContent"
+              slot-scope="context"
+            >
+              <div
+                class="questPopover"
+              >
+                <h4 class="popover-content-title">
+                  {{ context.item.text }}
+                </h4>
+                <questInfo :quest="context.item" />
+              </div>
+            </template>
+          </item>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-10 offset-1 text-center">
+          <span
+            v-once
+            class="no-quest-to-start"
+          >
+            <b>{{ $t('noQuestToStartTitle') }}</b> <br>
+            <span v-html="$t('noQuestToStart', { questShop: '/shops/quests' })"></span>
+          </span>
+        </div>
+      </div>
+    </div>
+    <div v-else>
+      <div v-if="questData" class="quest-combined-content">
+        <questDialogContent
+          :item="questData"
+          :group="group"
+          class="quest-detail"
+        />
+        <quest-rewards :quest="questData" class="mt-4" />
+      </div>
+      <div
+        v-if="!groupHasQuest"
+        class="text-center"
+      >
+        <button
+          class="btn btn-primary mt-0 invite-btn"
+          :disabled="!Boolean(selectedQuest) || loading"
+          @click="questInit()"
+        >
+          {{ $t('inviteToPartyOrQuest') }}
+        </button>
+      </div>
+      <div v-if="fromSelectionDialog"
```

Comment:
These indents are weird, would be just a nudge more readable if they were consistent with the rest of the code style in this file!